.github/workflows/release-windows.yml

```diff
@@ -45,17 +45,60 @@ jobs:
                   cargo clean
                   cargo build ${{ matrix.flavor == 'release' && '--release' || '' }} --target ${{ matrix.rust_target_prefix }}-pc-windows-msvc
 
-            - name: Create archive
+            - name: Upload unsigned executable for signing
```

Comment:
I think you can remove `DEBUG_FLAG` from `build_windows:`?

Reply:
yes! https://github.com/posit-dev/ark/pull/985/commits/3156b9737961ade19b60c639771176626743f0a6